green_cli/common.py

```diff
@@ -532,6 +532,35 @@ def mnemonic(file_, mnemonic):
         mnemonic = fileinput.input(mnemonic).readline()
     DefaultAuthenticator(context.options).set_mnemonic(mnemonic)
 
+@green.command()
+@click.option('--method', default="GET", expose_value=False, callback=details_json)
+@click.option('--urls',  default="",multiple=True, expose_value=False, callback=details_json)
```

Comment:
Would prefer singular '--url' for consistency with '--addressee'

Reply:
done, ta